### PR TITLE
Use SPACE_HOST instead of SPACE_SUBDOMAIN

### PR DESCRIPTION
**Pull request by XciD**
Side effect of https://github.com/huggingface/spaces-app-manager/issues/932

### CODE REVIEW

File: docs/hub/spaces-overview.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ In some cases, you might be interested in having programmatic access to the Spac
 * `SPACE_REPO_NAME`: i-like-flan
 * `SPACE_TITLE`: I Like Flan (specified in the README file)
 * `SPACE_ID`: `osanseviero/i-like-flan`
-* `SPACE_SUBDOMAIN`: `osanviero-i-like-flan` (the space's embed url is https://osanviero-i-like-flan.hf.space)
+* `SPACE_HOST`: `osanviero-i-like-flan.hf.space`
```
Comment (osanseviero) [inline docs/hub/spaces-overview.md:105]: :D 
```suggestion
* `SPACE_HOST`: `osanseviero-i-like-flan.hf.space`
```

File: docs/hub/spaces-oauth.md [hunk 2]
```diff
@@ -51,10 +51,10 @@ As for any other environment variable, you can use them in your code by using `o
 
 The allowed redirect URIs for your OAuth app are:
 
-- `https://{SPACE_SUBDOMAIN}.hf.space/auth/callback`
-- `https://{SPACE_SUBDOMAIN}.hf.space/login/callback`
+- `https://{SPACE_HOST}/auth/callback`
+- `https://{SPACE_HOST}/login/callback`
 
-Note that `SPACE_SUBDOMAIN` is also [available](https://huggingface.co/docs/hub/spaces-overview#helper-environment-variables) as an environment variable.
+Note that `SPACE_HOST` is also [available](https://huggingface.co/docs/hub/spaces-overview#helper-environment-variables) as an environment variable.
```
Comment (osanseviero) [inline docs/hub/spaces-oauth.md:57]: Is this change backwards compatible?

**Review by osanseviero — APPROVED** [review]
Thanks! :fire:

File: docs/hub/spaces-oauth.md (hunk 2 above)
Comment (XciD) [reply docs/hub/spaces-oauth.md:57]: The variable is already in prod if it's the question

Comment (julien-c) [reply docs/hub/spaces-oauth.md:57]: yeah but is `SPACE_SUBDOMAIN` still exposed? to not break any existing Space that might use it

Comment (julien-c) [reply docs/hub/spaces-oauth.md:57]: (though it seems not to be used at least in public repos: https://huggingface.co/search/full-text?q=SPACE_SUBDOMAIN&type=space )

**Review by julien-c — APPROVED** [review]

File: docs/hub/spaces-oauth.md (hunk 2 above)
Comment (XciD) [reply docs/hub/spaces-oauth.md:57]: Will remove it one month after this PR is merge if it's not used on any repo

Comment (julien-c) [reply docs/hub/spaces-oauth.md:57]: yep perfect

**Pull request merged**